src/EnergyPlus/DataDaylighting.hh

```diff
@@ -63,27 +62,31 @@
 
 namespace EnergyPlus {
 
-namespace DataDaylighting {
+namespace Dayltg {
 
     // Two kinds of reference points: used directly in daylighting, used to show illuminance map of zone
     constexpr int MaxMapRefPoints(2500); // Maximum number of Illuminance Map Ref Points
 
-    enum class SkyType : int
+    enum class SkyType 
     {
         Invalid = -1,
-        Clear = 1,
+        Clear,
         ClearTurbid,
         Intermediate,
         Overcast,
         Num
     };
 
+    struct Illums {
```

Comment:
A `struct` to hold the sky illum array. May expand to hold the sun illum components later.